src/lib/MVC/Symfony/Templating/Twig/Extension/RenderExtension.php

```diff
@@ -59,12 +60,27 @@ public function getFunctions(): array
         ];
     }
 
-    public function render(ValueObject $valueObject, array $options = []): string
+    /**
+     * @param \Ibexa\Contracts\Core\Repository\Values\ValueObject|\Ibexa\Contracts\Core\Repository\Values\Content\ContentAwareInterface $object
```

Comment:
Surprisingly enough, if we use:
```suggestion
     * @phpstan-param \Ibexa\Contracts\Core\Repository\Values\ValueObject|\Ibexa\Contracts\Core\Repository\Values\Content\ContentAwareInterface $object
     *
```
then PHPStorm stops reporting its internal static analysis issue
> Argument type does not match the declared 

As you know, I don't like false negatives, so I'd prefer this workaround.